dashboard/src/components/OperatorList/OperatorsNotSupported.v2.tsx

```diff
@@ -6,16 +6,17 @@ import * as url from "shared/url";
 
 interface IOperatorNotSupportedProps {
   namespace: string;
+  kubeappsCluster: string;
 }
 
 function OperatorNotSupported(props: IOperatorNotSupportedProps) {
   return (
     <Alert theme="warning">
-      <h5>Operators are supported on the default cluster only</h5>
+      <h5>Operators are supported on the cluster on which Kubeapps is installed only</h5>
       <p>
         Kubeapps' Operator support enables the addition of{" "}
-        <Link to={url.app.operators.list("default", props.namespace)}>
-          operators on the default cluster only
+        <Link to={url.app.operators.list(props.kubeappsCluster, props.namespace)}>
```

Comment:
Did I miss this the other day, or was it landed afterward? Thanks!

Reply:
No, I noticed that the OperatorNotSupported component changed while working on this and added it to this PR.